.github/workflows/CI.yml

```diff
@@ -1,13 +1,16 @@
-name: Continuous Integration
-on: 
+name: CI
+on:
   push:
+    branches:
+      - main
```

Comment:
why only limit CI to branch `main`? (release stage is already limited)

Reply:
It's not just limited to `main` :)

It also runs for PRs!

Reply:
yes the CI will run on *PR*s, but from what i can tell, it wont run on other branches in this repository